apple/Sources/Sargon/Extensions/Methods/Profile/Factor/PasswordFactorSource+Functions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  File.swift
+//
+//
+//  Created by Alexander Cyon on 2024-06-02.
```

Comment:
Created by Dan on 2024-11-29 :)

Reply:
we should remove those headers with swiftformat :) and we should format swift code like @matiasbzurovski added in som ancient PR.

in a separate PR..

Reply:
Was that PR merged though?

Reply:
I can take care of this now